### PR TITLE
Add passes paramter

### LINKED ISSUE #28: Supports arg num_passes when paddlecloud submit ...
`num_passes` will be set as an environment variable `PADDLE_INIT_NUM_PASSES`, we can startup trainer process in v1:
```bash
paddle train \
    --num_passes=${PADDLE_INIT_NUM_PASSES}
```
For V2 API, `paddle.init()` will load all environment variable wich the prefix is `PADDLE_INIT`.

### LINKED ISSUE #79: Trainer process startup failed
The trainer process can not be started correctly, the error logs:
```bash
ERROR: illegal value '' specified for int32 flag 'num_passes'
ERROR: illegal value '' specified for bool flag 'use_gpu'
```

Maybe #28 is a related issue.

### PR DESCRIPTION
**Pull request by Yancey1989**
Fixed #79
Fixed #28

### CODE REVIEW

File: go/paddlecloud/submit.go [hunk 1]
```diff
@@ -54,6 +55,7 @@ func (p *SubmitCmd) SetFlags(f *flag.FlagSet) {
 	f.StringVar(&p.PSMemory, "psmemory", "1Gi", "Parameter server momory resource. Defaults to 1Gi.")
 	f.StringVar(&p.Entry, "entry", "paddle train", "Command of starting trainer process. Defaults to paddle train")
 	f.StringVar(&p.Topology, "topology", "", "Will Be Deprecated .py file contains paddle v1 job configs")
+	f.IntVar(&p.Passes, "passes", 1, "Train so many pass for a job")
```
Comment (typhoonzero) [inline go/paddlecloud/submit.go]: Pass count for training job

File: paddlecloud/paddlejob/paddle_job.py [hunk 2]
```diff
@@ -71,6 +73,11 @@ def get_env(self):
         envs.append({"name":"PADDLE_INIT_PORTS_NUM",            "value":str(self._ports_num)})
         envs.append({"name":"PADDLE_INIT_PORTS_NUM_FOR_SPARSE", "value":str(self._ports_num_for_sparse)})
         envs.append({"name":"PADDLE_INIT_NUM_GRADIENT_SERVERS", "value":str(self._num_gradient_servers)})
+        envs.append({"name":"PADDLE_INIT_NUM_PASSES",           "value":str(self._passes)})
+        if self._gpu:
+            envs.append({"name":"PADDLE_INIT_USE_GPU", "value":str("true")})
```
Comment (typhoonzero) [inline paddlecloud/paddlejob/paddle_job.py]: Must be 0 or 1?

File: go/paddlecloud/submit.go (hunk 1 above)
Comment (Yancey1989) [inline go/paddlecloud/submit.go]: Done.

File: paddlecloud/paddlejob/paddle_job.py (hunk 2 above)
Comment (Yancey1989) [inline paddlecloud/paddlejob/paddle_job.py]: Done.

File: go/paddlecloud/submit.go [hunk 3]
```diff
@@ -54,6 +55,7 @@ func (p *SubmitCmd) SetFlags(f *flag.FlagSet) {
 	f.StringVar(&p.PSMemory, "psmemory", "1Gi", "Parameter server momory resource. Defaults to 1Gi.")
 	f.StringVar(&p.Entry, "entry", "paddle train", "Command of starting trainer process. Defaults to paddle train")
 	f.StringVar(&p.Topology, "topology", "", "Will Be Deprecated .py file contains paddle v1 job configs")
+	f.IntVar(&p.Passes, "passes", 1, "Pass count for for training job")
```
Comment (typhoonzero) [inline go/paddlecloud/submit.go]: 两个for了。。。

Comment (Yancey1989) [inline go/paddlecloud/submit.go]: Done...

**Pull request merged**